### ISSUE TITLE
 Custom TextStyle

### ISSUE THREAD

**Issue by ldk123456** [MarcinusX/NumberPicker#62]
I hope can customize the textstyle and margin or padding of text

**Comment by Fox-Ahmed** [comment]
Me too
Is it any way to customize ????

**Comment by MaskyS** [comment]
At first, I forked the package and directly made changes to (my local) library's code.

But since NumberPicker gets the color from the Theme you use, an easier method would be to wrap your `NumberPicker` widget with a Theme widget, then modify the property of your choice. 
For instance, I wanted to change the color of the default integers in my list to white. 

If you look at https://github.com/MarcinusX/NumberPicker/blob/edc933be247db508136781c7518dc32da294e61a/lib/numberpicker.dart#L244 , you notice the style for that Text is copied from the `body1` TextStyle in your Theme

So here's what I did:

```
  @override
  Widget build(BuildContext context) {
    var theme = Theme.of(context);
    var body = theme.textTheme.body1.copyWith(color: Colors.white);
    var textTheme = theme.textTheme.copyWith(body1: body);
    theme = theme.copyWith(textTheme: textTheme);

   {....}
    Theme(
      data: theme,
      child: NumberPicker.integer(
        // Set your values
       ),
    )
```

Hope this helps! Maybe sometime in the future, we'll be able to pass styles as arguments instead :)

Thanks @MarcinusX for the awesome package!
